src/main/java/Duke.java

```diff
@@ -13,37 +19,56 @@ public static void main(String[] args) {
         System.out.println("What can I do for you today?");
         String line;
         Scanner in = new Scanner(System.in);
-        ArrayList<Task> list = new ArrayList<Task>();
+
+
         while (true) {
             line = in.nextLine();
             String arr[] = line.split(" ", 2);
             if (line.equals("bye")) {
                 System.out.println("Bye see you again!");
                 break;
             } else if (line.equals("list")) {
-                System.out.println("Here's your plan for a productive day!");
+                if (list.size() == 0) {
+                    System.out.println("Nothing to do for now!Take a break!");
+                } else {
+                    System.out.println("Here's your plan for a productive day!");
+                }
                 for (int i = 0; i < list.size(); i++) {
-                    System.out.println(String.format("%d.[%s] %s", i + 1,
-                            list.get(i).getStatusIcon(), list.get(i).getDescription()));
+                    System.out.println(String.format("%d.[%s] [%s] %s", i + 1,
+                            list.get(i).getTypeIcon(), list.get(i).getStatusIcon(), list.get(i).getDescription()));
                 }
             } else if (arr[0].equals("mark")) {
                 System.out.println("You are crushing it,1 task down!");
                 Task.markAsDone(list.get(Integer.parseInt(arr[1]) - 1));
                 int i = Integer.parseInt(arr[1]) - 1;
-                System.out.println(String.format(" [%s] %s",
-                        list.get(i).getStatusIcon(), list.get(i).getDescription()));
+                System.out.println(String.format(" [%s] [%s] %s",
+                        list.get(i).getTypeIcon(), list.get(i).getStatusIcon(), list.get(i).getDescription()));
 
             } else if (arr[0].equals("unmark")) {
                 System.out.println("I have unchecked it for you");
                 Task.markAsNotDone(list.get(Integer.parseInt(arr[1]) - 1));
                 int i = Integer.parseInt(arr[1]) - 1;
-                System.out.println(String.format(" [%s] %s",
-                        list.get(i).getStatusIcon(), list.get(i).getDescription()));
+                System.out.println(String.format(" [%s] [%s] %s",
+                        list.get(i).getTypeIcon(), list.get(i).getStatusIcon(), list.get(i).getDescription()));
+
+            } else if (arr[0].equals("todo")) {
+                addTasks(new Todo(arr[1]));
+                System.out.println("Added!");
+                System.out.println(String.format(" [%s] [%s] %s", 'T', " ", arr[1]));
+                System.out.println("Now you have " + list.size() + " tasks in your list");
+            } else if (arr[0].equals("deadline")) {
+                String[] parts = arr[1].split("/");
+                addTasks(new Deadline(parts[0], parts[1]));
```

Comment:
```suggestions
Deadline[] deadline = new Deadline(parts[0], parts[1]);
addTasks(deadline);
```
perhaps you should create a variable first before adding to the method for better readability?